il/data/analyzer.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+import pandas as pd
+
+from tsdata.utils import download_and_unzip_data, get_datafile_path
+from il.data.importer import load_csv
+
+logger = logging.getLogger(__name__)
+
+COLUMNS_TO_ANALYZE = (
+    'agencycode', 'agencyname', 'Gender', 'Race', 'Search',
+    'Contraband', 'StopPurpose', 'year'
+)
+
+
+def stats_for_state_landing_page(stops, report):
+
+    def get_agency_name(n):
+        """
+        horribly slow?
```

Comment:
It isn't run during steady state.  I expect that there is a better way to do it but I couldn't find it in a reasonable period of time, and I didn't want anybody to take this as an appropriate example of how to perform this in Pandas.  I should fix the comment accordingly.